prisma/schema.prisma

```diff
@@ -31,8 +31,34 @@ model User {
   role          Role      @default(TEACHER)
   status        Status    @default(INVITED)
   numOfAbsences Int       @default(10)
-  absences      Absence[] @relation("absentTeacher")
-  substitutes   Absence[] @relation("substituteTeacher")
+  absences      Absence[] @relation("absentTeacher") // corrected relation for absences
+  substitutes   Absence[] @relation("substituteTeacher") // relation for substitute teacher absences
```

Comment:
```suggestion
  absences      Absence[] @relation("absentTeacher")
  substitutes   Absence[] @relation("substituteTeacher")
```
It seems like these 2 lines are the same as before, not sure if the comment is relevant